assets/css/style.css

```diff
@@ -0,0 +1,4 @@
+img {
+    width: 100px;
+    height: 100px;
+}
```

Comment:
when scaling down images you need to pick either `width` or `height` but not both. any image with aspect ration other than 1:1 will be distorted